src/NUnitFramework/tests/Assertions/LowTrustFixture.cs

```diff
@@ -61,7 +61,7 @@ public void AssertEqualityInLowTrustSandBox()
             });
         }
 
-        //[Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
+        [Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
         // TODO: This test fails to run correctly when Assert.Throws
         // attempts to create an isolated TestExecutionContext.
         // The TestExecutionContext class would have to be modified
```

Comment:
These comments are no longer relevant and should be removed as to not confused...